core/trino-main/src/main/java/io/trino/execution/buffer/CompressionCodec.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.execution.buffer;
+
+public enum CompressionCodec
+{
+    NONE, LZO, SNAPPY, LZ4, ZSTD
```

Comment:
What is the use of `NONE` if we have separate flag to enable/disable compression

Reply:
> What is the use of `NONE` if we have separate flag to enable/disable compression

You are right, NONE is useless in this case, I copied the enum from `io.trino.orc.metadata.CompressionKind` at the beginning~ I have deleted the NONE value